README.md

```diff
@@ -8,7 +8,7 @@ WallGo is open-source scientific software for computing the bubble wall speed fo
 
 ## Status
 
-**Version** v0.1.0 (alpha)
+[![Version](https://img.shields.io/github/v/tag/Wall-Go/WallGo?label=Version)](https://github.com/Wall-Go/WallGo/tags/)
```

Comment:
Shouldn't this go to the `latest` release?

Reply:
I don't think we have any "releases" yet for this repo.

Reply:
If you can find a better url to point to, please suggest it.

Reply:
Right, let's update this when we have a release.